src/fluxgui/exceptions.py

```diff
@@ -1,18 +1,22 @@
 __all__ = ['DirectoryCreationError', 'FileNotFoundError',
-        'XfluxError', 'MethodUnavailableError']
+           'XfluxError', 'MethodUnavailableError']
+
```

Comment:
It would be much easier to review this PR without all of these whitespace changes. Better to not change the style of existing code, if you don't have some other reason to edit it, even if the formatting seems weird.

Reply:
> It would be much easier to review this PR without all of these whitespace changes. Better to not change the style of existing code, if you don't have some other reason to edit it, even if the formatting seems weird.

I will fix the whitespaces